models/dalec/R/write.configs.dalec.R

```diff
@@ -71,17 +71,40 @@ convert.samples.DALEC <- function(trait.samples) {
     names(trait.samples)[which(names(trait.samples) == "som_respiration_rate")] <- "t9"
   }
   
-  ### INITIAL CONDITIONS
-  
-  # cf0 initial canopy foliar carbon (g/m2)
-  # cw0 initial pool of woody carbon (g/m2)
-  # cr0 initial pool of fine root carbon (g/m2)
-  # cl0 initial pool of litter carbon (g/m2)
-  # cs0 initial pool of soil organic matter and woody debris carbon (g/m2)
-  
   return(trait.samples)
 } # convert.samples.DALEC
 
+####partition_roots: function to split root_carbon_content into fine and coarse roots by rtsize dimension at the .002 m threshold
+partition_roots <- function(roots, rtsize){
```

Comment:
There doesn't seem to be anything DALEC-specific here. Is this something that could be moved to data.land and used by other models?